src/teleop/gui/src/components/IMUCalibration.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+<div class = "wrap">
+    <div class="table">
+        <table class="tableFormat" style="undefined;table-layout:fixed;border=0">
+            <caption>IMU Calibration</caption>
+            <colgroup>
+            <col style="width:80px">
+            <col style="width:80px">
+            <col style="width:80px">
+            <col style="width:80px">
+        </colgroup>
+        <thead>
+            <tr>
+                <td class="tableElement">Mag</td>
+                <td class="tableElement">Accel</td>
+                <td class="tableElement">Gyro</td>
+                <td class="tableElement">System</td>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <!--<td><span class='led' ,class="{{IMUCalibration[0].color}}"></span></td>-->
+                <td><span v-bind:class='["led",magnetometer_calibration_color?"green":"red"]'></span></td>
+                <td><span v-bind:class='["led",accelerometer_calibration_color?"green":"red"]'></span></td>
+                <td><span v-bind:class='["led",gyroscope_calibration_color?"green":"red"]'></span></td>
+                <td><span v-bind:class='["led",system_calibration_color?"green":"red"]'></span></td>
+            </tr>
+        </tbody>
+        <tbody>
+            <tr>
+                <td class="tableElement">{{magnetometer_val}}</td>
+                <td class="tableElement">{{accelerometer_val}}</td>
+                <td class="tableElement">{{ gyroscope_val }}</td>
+                <td class="tableElement">{{system_val }}</td>
+            </tr>
+        </tbody>
+        </table>
+    </div>
+</div>
+</template>
+
+<script>
+import ROSLIB from "roslib"
+const calibration_limit=3
+
+export default {
+    data() {
+        return {
+            system_calibration_color : false,
+            gyroscope_calibration_color : false,
+            accelerometer_calibration_color : false,
+            magnetometer_calibration_color : false,
```

Comment:
I think the code may be more readable without these four boolean values. Up in the html template area you could just let the value in the ternary operator be `<val> == calibration_limit`